src/sidebar/panel.html

```diff
@@ -5,6 +5,9 @@
   <title>Firefox Notes</title>
   <link href="vendor/quill.snow.css" rel="stylesheet">
   <link href="styles.css" rel="stylesheet">
+
+	<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Indenting on these 2 new lines looks a bit off...

Also, possible to bundle the font family into the add-on? It seems unfortunate to have to proxy everything from the 3rd party fonts.googleapis.com site.

/cc @vladikoff